Laravel-Backpack/demo
# Add example for all 50+ column types (both CRUD and PRO)

## WHY

### BEFORE - What was wrong? What was happening before this PR?

We only had examples for 20+ column types, because that's all we had.

### AFTER - What is happening after this PR?

There are two PRs that bring feature-parity between fields and columns. Basically, each CRUD and PRO field now has a corresponding column type, with the same name:
- https://github.com/Laravel-Backpack/CRUD/pull/4675
- https://github.com/Laravel-Backpack/PRO/pull/91

This PR makes it easy to all those column types that those PRs add.

## HOW

### How did you achieve that, in technical terms?

Added new CrudController, `ColumnMonsterCrudController`, that basically uses fields definitions to add columns. That way, we also test that the syntax for fields works for its corresponding column too.

### Is it a breaking change or non-breaking change?

Non-Breaking change

### How can we test the before & after?

- checkout this PR on the demo
- go to `vendor/backpack/crud` and checkout PR https://github.com/Laravel-Backpack/CRUD/pull/4675
- go to `vendor/backpack/pro` and checkout PR https://github.com/Laravel-Backpack/PRO/pull/91
- in your browser, go to `/admin/column-monster` and play around - see all columns work
